LuaUI/widgets/chobby/components/chat_windows.lua

```diff
@@ -342,3 +364,67 @@ function ChatWindows:GetPrivateChatConsole(userName)
 
     return privateChatConsole
 end
+
+function ChatWindows:JoinChannel()
```

Comment:
This function should be called Create/Spawn JoinChannel Dialogue/Window/Popup, e.g. CreateJoinChannelWindow

(I'm not sure I'm entirely consistent with the create/spawn and dialogue/window/popup naming either)